launch-scripts/launch_gapbs.py

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env python3
+
+#This is a job launch script for boot tests
+
+import os
+import sys
+from uuid import UUID
+
+from gem5art.artifact.artifact import Artifact
+from gem5art.run import gem5Run
+from gem5art.tasks.tasks import run_gem5_instance
+
+packer = Artifact.registerArtifact(
+    command = '''wget https://releases.hashicorp.com/packer/1.4.3/packer_1.4.3_linux_amd64.zip;
+    unzip packer_1.4.3_linux_amd64.zip;
+    ''',
+    typ = 'binary',
+    name = 'packer',
+    path =  'disk-image/packer',
+    cwd = 'disk-image',
+    documentation = 'Program to build disk images. Downloaded sometime in August from hashicorp.'
+)
+
+experiments_repo = Artifact.registerArtifact(
+    command = 'git clone https://your-remote-add/gapbs-test.git',
+    typ = 'git repo',
+    name = 'gapbs-test',
+    path =  './',
+    cwd = '../',
+    documentation = 'main experiments repo to run GAPBS with gem5'
+)
+
+gem5_repo = Artifact.registerArtifact(
+    command = 'git clone https://gem5.googlesource.com/public/gem5',
```

Comment:
Are you running with gem5 19?

Reply:
Yes, but I also ran it with gem5 20. I just wanted to be consistent with the other experiments.